### DISCUSSION TITLE
🎃 Storacha Halloween Hackathon 2024 – Build with Decentralized Storage and Win $6666 💀💻

### DISCUSSION THREAD

**Discussion by hakierka** [storacha/w3up#1556] [General]
![Haloween-Card-9_30_24](https://github.com/user-attachments/assets/954da4af-dc39-48f3-84c0-22c3297afb6e)
Hey developers! 👋

We’re hosting the Storacha Halloween Hackathon 2024, and it’s the perfect opportunity to build something cool using decentralized storage technology. Whether you’re passionate about blockchain, Web3, or just love hackathons, this event is for you! 👻

Here are the details:

**Prizes**: $6666 in total! 🏆
**Key Dates:**
**Registration Deadline:** October 13, 2024
**Coding Period:** October 16-22
**Submission Deadline:** October 27
**Where:** Everything’s happening online via Discord and TAIKAI.
We’re challenging devs to get creative with Storacha’s decentralized storage technology. Think dApps, games, or even new ways to store “skeletons” (or data 💀). If you’re interested in using decentralized tech to build something exciting, join us!

**🔗 Sign up here:** [Storacha Halloween Hackathon](https://taikai.network/storacha-network/hackathons/halloween)

**See you on Discord:**
 [discord.gg/ppnTA9uqWN](https://discord.gg/ppnTA9uqWN)
Can’t wait to see what you build!

**Comment by hannahhoward** [comment]
Exciting!

**Comment by tinc-x** [comment]
where can I see the Hackathon result?

**Reply by travis** [reply]
We published them on our blog: https://medium.com/@storacha/storacha-halloween-hackathon-2024-unveiling-the-winners-08d3a15f8338
